games-util/fteqcc/fteqcc-20250126.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Quake-C compiler"
+HOMEPAGE="https://www.fteqcc.org/"
+MY_COMMIT="f767d952e3ad8bbcb52f1cd6e2e36a47e3dbaa87"
```

Comment:
Maybe add a reference in a comment to https://bugs.gentoo.org/859307#c3 in case people wonder why we're doing snapshots, but I guess it's not that important (one can figure it out from the lack of tags).